### PR TITLE
new release notes

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/cloud-cost-management.md [hunk 1]
```diff
@@ -18,12 +18,31 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
 :::
 
+
 ## October 2024
 
 ### Version 1.32.1
 
 #### New features and enhancements
 
+- AWS Payer Account ID Field Added: We’ve introduced the AWS Payer Account ID in the AWS fields dropdown for Perspectives and CC Rules. Only account IDs will be shown, excluding account names. [CCM-19843]
```
Comment (Rohit-Kaliki) [inline release-notes/cloud-cost-management.md:28]: Can we call out CC as cost categories

File: release-notes/cloud-cost-management.md [hunk 2]
```diff
@@ -18,12 +18,31 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+- Increased Connector Limit for GCP: We raised the default limit to 6,000 for GCP cross combinations (target account * target region) to prevent unannounced system loads, aligning with the 200 account and 30 region limits. [CCM-19838]
```
Comment (Rohit-Kaliki) [inline release-notes/cloud-cost-management.md:30]: Let's add that this is got Governance

File: release-notes/cloud-cost-management.md [hunk 3]
```diff
@@ -18,12 +18,31 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+- ‘Select All’ Option for Filtered Search: A new ‘Select All’ option allows users to select an entire filtered list, automatically updating based on the filtered options. [CCM-19794]
```
Comment (Rohit-Kaliki) [inline release-notes/cloud-cost-management.md:32]: Again for what feature?

File: release-notes/cloud-cost-management.md [hunk 4]
```diff
@@ -18,12 +18,31 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+- Date Filter Sync Issue on Evaluation Details Page: Fixed a date filter sync issue displaying "undefined - undefined" on load. Users previously had to manually select a date range, which is now resolved. [CCM-19785]
+
+- Fixed ECS Metrics Chart Issue: CPU recommendations in ECS metrics charts previously displayed incorrect graphs, and CPU usage showed as 0. This has now been corrected. [CCM-19715]
+
+- Ad-Hoc Enforcement Runs: Users can now dry run or execute enforcements directly from the Enforcements list or details page, improving control over ad-hoc enforcement. [CCM-19438]
```
Comment (Rohit-Kaliki) [inline release-notes/cloud-cost-management.md:38]: Can we move this to the top above bugs

Comment (Rohit-Kaliki) [reply release-notes/cloud-cost-management.md:38]: This should be a feature

**Review by Rohit-Kaliki — CHANGES_REQUESTED** [review]

File: release-notes/cloud-cost-management.md (hunk 1 above)
Comment (joeyouss) [reply release-notes/cloud-cost-management.md:28]: done

File: release-notes/cloud-cost-management.md (hunk 2 above)
Comment (joeyouss) [reply release-notes/cloud-cost-management.md:30]: done

File: release-notes/cloud-cost-management.md (hunk 3 above)
Comment (joeyouss) [reply release-notes/cloud-cost-management.md:32]: done

File: release-notes/cloud-cost-management.md (hunk 4 above)
Comment (joeyouss) [reply release-notes/cloud-cost-management.md:38]: done

File: release-notes/cloud-cost-management.md [hunk 5]
```diff
@@ -18,12 +18,33 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
 ## October 2024
 
 ### Version 1.32.1
 
 #### New features and enhancements
 
+- Ad-Hoc Enforcement Runs: Users can now dry run or execute enforcements directly from the Enforcements list or details page in Asset Governance, improving control over ad-hoc enforcement. [CCM-19438]
+
+- New Resource Name Column in Recommendations View: A new column, Resource Name (No drilldown) has been added to the Governance recommendations view. This allows users to group Governance Recommendations by a specific rule. [CCM-19922]
```
Comment (iamanmolm) [inline release-notes/cloud-cost-management.md:30]: “specific rule in Dashboards.”

**Review by iamanmolm — APPROVED** [review]

**Pull request merged**